### ISSUE TITLE
endSendingPlayMessages being called twice on dealloc

### ISSUE THREAD

**Issue by rclai** [rFlex/SCRecorder#343]
Is it intentional to call `endSendingPlayMessages` twice on `dealloc` [here](https://github.com/rFlex/SCRecorder/blob/3f39b323d2b02e0b013a59242185b33483994935/Library/Sources/SCPlayer.m#L53) and [here](https://github.com/rFlex/SCRecorder/blob/3f39b323d2b02e0b013a59242185b33483994935/Library/Sources/SCPlayer.m#L59)?

**Comment by rFlex** [comment]
I don't believe it's intentional, although the second time it's called it should be a no-op.
